common/element_handle.go

```diff
@@ -1042,7 +1042,8 @@ func (h *ElementHandle) Query(selector string) (*ElementHandle, error) {
 		return nil, fmt.Errorf("querying selector %q: %w", selector, err)
 	}
 	if result == nil {
-		return nil, fmt.Errorf("querying selector %q", selector)
+		h.logger.Debugf("ElementHandle:Query:result:nil", "sel:%q", selector)
```

Comment:
Left a debug log here which might be useful. Happy to remove if you think there's no value in it.

Reply:
Not sure if it will add much value, but no strong opinion against it, if you think it can help us in some cases, let's keep it.

Reply:
Removed in 54a09aeba83c7c2c376e617e3df328af059038f3